stdlib/asyncio/taskgroups.pyi

```diff
@@ -0,0 +1,17 @@
+import sys
+from types import TracebackType
+from typing import Any, Coroutine, Generator, TypeVar
+
+from .tasks import Task
+
+_T = TypeVar("_T")
+
+if sys.version_info >= (3, 11):
+    class TaskGroup:
+        def __init__(self, name: str | None = ...) -> None: ...
+        def get_name(self) -> str: ...
+        async def __aenter__(self) -> TaskGroup: ...
+        async def __aexit__(self, et: type | None, exc: BaseException | None, tb: TracebackType | None) -> None: ...
```

Comment:
```suggestion
        async def __aexit__(self, et: type[BaseException] | None, exc: BaseException | None, tb: TracebackType | None) -> None: ...
```

Reply:
Heh. I copied that literally from create_taks() in task.pyi. :-)